meta-resin-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -137,6 +138,10 @@ if [[ -n "${INTERNAL_DEVICE_BOOTLOADER_CONFIG}" ]]; then
             fail "Failed to mount disk labeled as 'resin-boot'."
         fi
         cp $EXTERNAL_DEVICE_BOOT_PART_MOUNTPOINT/$INTERNAL_DEVICE_BOOTLOADER_CONFIG $INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT/$INTERNAL_DEVICE_BOOTLOADER_CONFIG_PATH
+
+        # Copy custom splash dir
+        cp -r $EXTERNAL_DEVICE_BOOT_PART_MOUNTPOINT/$SPLASH_DIRNAME $INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT/$SPLASH_DIRNAME
+
```

Comment:
I think that for doing this we need some changes because doing it as above you add the splash only when INTERNAL_DEVICE_BOOTLOADER_CONFIG is set. I think we would want to have the splash copied regardless if this variable is set or not.